### ISSUE TITLE
[stable/stolon] keeper no db assigned?

### ISSUE THREAD

**Issue by JWDobken** [helm/charts#18055]
I followed the explanation of [Denis Isaev](https://disaev.me/p/how-i-have-set-up-ha-postgresql-on-kubernetes/) on how to configure Stolon as a HB Postgres database. All pods are running, but when I connect to any of the keepers I will get:

> server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.

The logs of all the keepers all give the following output (`kubectl logs -n postgres postgres-stolon-keeper-1`):

```sh
2019-10-16T20:10:01.057Z        INFO    cmd/keeper.go:982       no db assigned
2019-10-16T20:10:01.813Z        ERROR   cmd/keeper.go:641       cannot get configured pg parameters     {"error": "dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory"}
2019-10-16T20:10:04.314Z        ERROR   cmd/keeper.go:641       cannot get configured pg parameters     {"error": "dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory"}
```

So it complains no db is assigned and cannot get configured pg?

Kubernetes: v1.14.6-gke.1
Chart: version: 1.2.0
Helm version: v2.14.3

**Comment by obeyler** [comment]
same issue

**Comment by flexlee** [comment]
same issue

Kubernetes: v1.14.7-gke.23
Chart: stolon-1.4.0
Helm: 2.16.1

**Comment by flexlee** [comment]
similar to #18884

**Comment by flexlee** [comment]
#18915 seem to fix the problem

**Comment by obeyler** [comment]
yes it does. As workaround you can also do as I do... I stay on the  1.3.0 version :-)

**Comment by JWDobken** [comment]
Problem is solved for version 1.5.5
